api/Report.json

```
@@ -82,6 +82,331 @@
+              "tags": [
+                "web-features:reporting"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": "69"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": false,
+                "deprecated": false
+              }
+            }
+          },
+          "disposition": {
+            "__compat": {
+              "description": "`disposition` in `coep` report body",
+              "spec_url": "https://html.spec.whatwg.org/multipage/browsers.html#embedder-policy-checks",
+              "tags": [
+                "web-features:reporting"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": "69"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "79",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "dom.reporting.enabled",
+                      "value_to_set": "true"
+                    }
+                  ]
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "16.4"
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "type": {
+            "__compat": {
+              "description": "`type` in `coep` report body",
+              "spec_url": "https://html.spec.whatwg.org/multipage/browsers.html#embedder-policy-checks",
+              "tags": [
+                "web-features:reporting"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": "69",
+                  "notes": "Includes support for non-standard type `coep`"
```

FYI this takes a non standard option `corp` (, which is misprinted here as `coep`)
I think I should make this yet one lower subfeature for this case, but ONLY include the invalid values.
My reasoning is that when `corp` option is no longer supported, we'd need to track the change.

Thoughts?

```suggestion
                  "notes": "Includes support for non-standard type `corp`"
```